### PR TITLE
New tableview

### PR DESCRIPTION
**Pull request by axel-klinger**
Reorder of library entries in a table with country and city based on original data

### CODE REVIEW

**Comment by stweil** [comment]
Nice. With [GitHub-sort-content](https://github.com/Mottie/GitHub-userscripts/wiki/GitHub-sort-content) it is even possible to sort the table as desired.

I think we can use this as an intermediate solution.

For the future I imagine all current data (plus additional information) in a JSON file `BibsOnGitHub.json`, a Python script which creates nice looking tables from that data, optionally adding more information (from GitHub, from Wikidata, ...), optionally using native or English names, and much more.

A prebuilt list in markdown format made with that script (similar to the one here) could then be added to the repository as a convenience solution for end users.

File: list.md [hunk 1]
```diff
@@ -7,175 +7,68 @@ librarians.
 Library institutions, journals, and projects
 ---------------------------------------------
 
-Niedersächsische Staats- und Universitätsbibliothek Göttingen (SUB Göttingen, SUBUGOE)
-[https://github.com/subugoe](https://github.com/subugoe)
+| Country     | City            | Organisation                                                                                                          | Repositories |
```
Comment (zuphilip) [inline list.md]: Until we have any content I would delete the column "Repositories".

File: list.md [hunk 2]
```diff
@@ -7,175 +7,68 @@ librarians.
...
+| Luxembourg  |                 | [National Library of Luxembourg](https://github.com/natliblux)                                                        |              |
+| Netherlands |                 | [National Library of the Netherlands / Research](https://github.com/KBNLresearch)                                     |              |
+| Norway      | Bergen          | [University of Bergen Library (Norway)](https://github.com/ubbdst)                                                    |              |
+| Norway      | Oslo            | [Digital Development Team at Oslo Public Library](https://github.com/digibib)                                         |              |
+| Norway      | Oslo            | [University of Oslo Library (Norway)](https://github.com/scriptotek)                                                  |              |
+| Norway      |                 | [NTNU University Library (Norway)](https://github.com/NTNUUB)                                                         |              |
+| Norway      |                 | [BI Norwegian Business School Library (Norway)](https://github.com/bisentralen)                                       |              |
```
Comment (zuphilip) [inline list.md]: We can delete now the country in the parenthesis. Also for some of the other lines.

File: list.md [hunk 3]
```diff
@@ -7,175 +7,68 @@ librarians.
...
+| Germany     | Göttingen       | [Gemeinsamer Bibliotheksverbund (GBV Göttingen)](https://github.com/gbv)                                              |              |
+| Germany     | Hamburg         | [Staats- und Universitätsbibliothek Hamburg](https://github.com/subhh)                                                |              |
+| Germany     | Hamburg         | [Universitätsbibliothek der TU Hamburg](https://github.com/tubhh)                                                     |              |
+| Germany     | Hamburg/Kiel    | [ZBW - Leibniz Information Centre for Economics](https://github.com/zbw)                                              |              |
+| Germany     | Hannover        | [Technische Informationsbibliothek (TIB), Hannover](https://github.com/TIBHannover)                                   |              |
+| Germany     | Hannover        | [Bibliothek der Hochschule Hannover 1](https://github.com/HsH-Bibliothek)                                             |              |
+| Germany     | Hannover        | [Bibliothek der Hochschule Hannover 2](https://github.com/hsh-bib)                                                    |              |
```
Comment (zuphilip) [inline list.md]: I wouldn't make two entries for the HsH-Bib but rather give both links in the same row. The labels with 1 and 2 are not really meaningful.

File: list.md [hunk 4]
```diff
@@ -7,175 +7,68 @@ librarians.
...
+| Norway      |                 | [BI Norwegian Business School Library (Norway)](https://github.com/bisentralen)                                       |              |
+| Switzerland |                 | [swissbib (Swiss Library Catalogue)](https://github.com/swissbib)                                                     |              |
+| Switzerland |                 | [project linked-swissbib](https://github.com/linked-swissbib)                                                         |              |
+| UK          | London          | [The British Library](https://github.com/britishlibrary)                                                              |              |
+| UK          | London                | [British Library Labs Project](https://github.com/BL-Labs)                                                            |              |
+| Sweden      |                 | [National Library of Sweden 1](https://github.com/Kungbib)                                                            |              |
+| Sweden      |                 | [National Library of Sweden 2](https://github.com/Libris)                                                            |              |
```
Comment (zuphilip) [inline list.md]: Dito. And isn't Libris more than the national library, some sort of union catalogue?

File: list.md [hunk 5]
```diff
@@ -390,3 +283,110 @@ David Zellhöfer (SBB Berlin)
 
 Philipp Zumstein (UB Mannheim)
 [https://github.com/zuphilip](https://github.com/zuphilip)
+Jan Frederik Maas (SUB Hamburg)
+[https://github.com/jfmaas](https://github.com/jfmaas)
+
+Martin Malmsten (National Library of Sweden)
+[https://github.com/marma/](https://github.com/marma/)
```
Comment (zuphilip) [inline list.md]: This are adding the persons again as a duplicate. Undo this.

**Comment by stweil** [comment]
@axel-klinger, I suggest not to add more commits until we have a common understanding about the next steps. For example I am not convinced that the list should be extended from a European to a world wide list.

**Comment by stweil** [comment]
One of the open questions is which primary data format (the one which is edited manually) is preferred. We currently have markdown, the pull request suggests CSV, and personally I'd prefer JSON.

**Comment by axel-klinger** [comment]
What is the problem with a world wide list?

Isn't CSV easier to edit than JSON? And it renders as a searchable table on GitHub. See [CSV](https://github.com/axel-klinger/BibsOnGitHub/blob/master/libraries.csv)

**Comment by stweil** [comment]
> What is the problem with a world wide list?

Let me answer with more questions.

- Who would maintain a world wide list? I think that only regional lists are maintainable.
- What is the purpose of any list? With whom do institutions and people usually work together?

However I could image scripts which collect data from other sources to create nice web pages with world wide information on github.io.

**Comment by stweil** [comment]
CVS looks nice on GitHub, but does not offer large flexibility. As soon as more (optional) fields get added, it no longer looks nice and maintenance becomes difficult.

**Comment by axel-klinger** [comment]
This is what I was looking for: [repositories.csv](https://github.com/axel-klinger/BibsOnGitHub/blob/master/repositories.csv) - a table of repositories where I can search for a component and institutions who are also working on. Example: type 'kited' or 'dspace' or 'doi' or 'apc' - and I am fine with results from out of Europe, if they match.

**Comment by axel-klinger** [comment]
I will close this pull request, as there are some very large files involved, that I maybe should not have committed ;-) Please feel free to take all information you like from [libraries.csv](https://github.com/axel-klinger/BibsOnGitHub/blob/master/libraries.csv) or [libraries.md](https://github.com/axel-klinger/BibsOnGitHub/blob/master/libraries.md).

**Pull request closed without merging**